### PR TITLE
Fix NatSpec json output for "@notice" and "@dev" tags on contract definitions

### PR DESCRIPTION
**Pull request by zone117x**
Fixes issue https://github.com/ethereum/solidity/issues/4446

### CODE REVIEW

**Review by ipreferpi — APPROVED** [review]
👌

**Review by ipreferpi — DISMISSED** [review]

File: libsolidity/interface/Natspec.cpp [hunk 1]
```diff
@@ -36,6 +36,10 @@ Json::Value Natspec::userDocumentation(ContractDefinition const& _contractDef)
 	Json::Value doc;
 	Json::Value methods(Json::objectValue);
 
+	auto dev = extractDoc(_contractDef.annotation().docTags, "notice");
```
Comment (axic) [inline libsolidity/interface/Natspec.cpp]: Why is the variable called `dev`? Shouldn't it be `notice`? Also below in the loop we use an explicit type (`string`).

**Comment by axic** [comment]
The natspec logic seems to be correct though (compared to the "spec").

File: libsolidity/interface/Natspec.cpp (hunk 1 above)
Comment (zone117x) [reply libsolidity/interface/Natspec.cpp]: Fixed!

**Comment by zone117x** [comment]
@axic Thanks for the feedback. Fixed the concerns you brought up. Are we good to merge now?

**Comment by chriseth** [comment]
Please add an entry into the changelog, in the "bugfixes" category.

**Comment by zone117x** [comment]
@chriseth Done!

**Review by chriseth — DISMISSED** [review]

**Pull request merged**